definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -12423,5 +12423,131 @@
     "CaseFieldID": "DateCaseNoLongerEligibleForDA",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolServiceMethod",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
```

Comment:
Do we know if this is going live in the next release (around the ~11th of Aug) or sometime later? If it's not going live soon I would personally suggest we keep it in the branch - cherry-pick to prod last time was a little tricky and took time :)

Reply:
These changes will not go as part of "Respondent Solicitor release" (11th august).

Should release these changes as part of "Solicitor - Personal Service"